pkg/server/server.go

```diff
@@ -386,8 +388,14 @@ func (s *ProxyServer) Proxy(stream client.ProxyService_ProxyServer) error {
 				close(stopCh)
 				return
 			}
+			if status.Code(err) == codes.Canceled {
+				klog.V(2).InfoS("Stream read from frontend cancelled", "userAgent", userAgent, "serverID", s.serverID)
```

Comment:
Do we want to move this inside the `err != nil` case below? We would avoid the duplicated code to send the error and close the channel